### ISSUE TITLE
Token Failed To Join ERROR

### ISSUE THREAD

**Issue by rew0rker** [H4cK3dR4Du/Discord-Token-Joiner#1]
![изображение_2023-07-19_152803503](https://github.com/H4cK3dR4Du/Discord-Token-Joiner/assets/74911702/2ac6f322-558a-4cd5-b5a3-af3ca6679eda)

**Comment by H4cK3dR4Du** [comment]
You understand?

**Comment by rew0rker** [comment]
Yes, now the script solves the captcha, but does not invite the account to the server
![изображение_2023-07-19_160228763](https://github.com/H4cK3dR4Du/Discord-Token-Joiner/assets/74911702/86df3b4d-025a-40f6-9a15-1e488d9c0e36)

**Comment by H4cK3dR4Du** [comment]
Ok i Will fix that soon

**Comment by rew0rker** [comment]
At you too the script now does not invite users?

**Comment by H4cK3dR4Du** [comment]
For me It does. Add me on discord: imh4ck3dr4du
